### PR TITLE
payment error handling

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#7641

*Description of changes:*
- payment error flow for search + registration flows

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/Dashboard.vue [hunk 1]
```diff
@@ -350,6 +358,7 @@ export default class Dashboard extends Vue {
     this.setSearchedValue('')
     this.setSearchResults(null)
     this.onAppReady(this.appReady)
+    console.log('here')
```
Comment (doug-lovett) [inline ppr-ui/src/views/Dashboard.vue]: Remove?

**Review by doug-lovett — APPROVED** [review]
Looks great, just one small comment about removing a log message.

**Review by jordiwes — APPROVED** [review]
LGTM!

File: ppr-ui/src/views/Dashboard.vue (hunk 1 above)
Comment (kialj876) [reply ppr-ui/src/views/Dashboard.vue]: haha oops! Yeah definitely

**Pull request merged**